lib/components/InputField/InputField.tsx

```diff
@@ -39,10 +39,10 @@ export const InputField = React.forwardRef<HTMLInputElement, InputFieldProps>(fu
         placeholder={placeholder ? `(${placeholder})` : undefined}
         autoComplete="off"
         aria-describedby={help ? helpId : undefined}
-        className="block w-full rounded-[10px] border-[5px] border-border-gray bg-white p-[11px] text-black outline-none placeholder:text-secondary-gray"
+        className="block w-full rounded-[8px] border border-border-gray bg-white p-[11px] text-black outline-none placeholder:text-secondary-gray"
```

Comment:
8px is the default border radius, so I think you should be able to just use `rounded`

Reply:
Fixed, now using the default value with just `rounded`.